dist/robots.cfg

```diff
@@ -27,11 +29,14 @@ RobotUA <<EOR
     BookmarkSync,
     bot,
     Builder,
+    bytespider,
+    Bytespider,
```

Comment:
We only need one of these because the match is case-insensitive as is seen in Vend::Config:

```perl
sub parse_list_wildcard {
    my $value = get_wildcard_list(@_,0);
    return '' unless length($value);
    return qr/$value/i;
}

sub parse_list_wildcard_full {
    my $value = get_wildcard_list(@_,1);
    return '' unless length($value);
    return qr/^($value)$/i;
}
```